packages/salesforce-adapter/src/filters/custom_object_instances_references.ts

```diff
@@ -82,10 +80,42 @@ const createWarnings = async (
   instancesWithEmptyIds: InstanceElement[],
   missingRefs: MissingRef[],
   illegalRefSources: Set<string>,
-  customObjectPrefixKeyMap: Record<string, string>,
   dataManagement: DataManagement,
   baseUrl?: string,
 ): Promise<SaltoError[]> => {
+  const createOmittedInstancesWarning = async (
+    originTypeName: string,
+    missingRefsFromOriginType: MissingRef[],
+  ): Promise<SaltoError> => {
+    const typesOfMissingRefsTargets: string[] = _(missingRefsFromOriginType)
+      .map(missingRef => missingRef.origin.field?.annotations[FIELD_ANNOTATIONS.REFERENCE_TO])
+      .filter(isDefined)
+      .uniq()
+      .value()
+    const numMissingInstances = _(missingRefsFromOriginType)
+      .map(missingRef => missingRef.targetId)
+      .uniq()
+      .value().length
```

Comment:
```suggestion
    const numMissingInstances = _.uniqBy(
    missingRefsFromOriginType,
    missingRef => missingRef.targetId
    ).length
```
Nit: Consider the following instead. Not a fan of using **map** just to count unique occurrences.

Reply:
Oh. I didn't know about `_.uniqBy` :)